ServiceFabric.PubSubActors/Helpers/IPublisherServiceHelper.cs

```diff
@@ -6,15 +6,22 @@ namespace ServiceFabric.PubSubActors.Helpers
 {
     public interface IPublisherServiceHelper
     {
+        /// <summary>
+        /// Publish a message.
+        /// </summary>
+        /// <param name="message"></param>
+        /// <param name="brokerServiceName">The name of a SF Service of type <see cref="BrokerService"/>.</param>
+        /// <returns></returns>
+        Task PublishMessageAsync(object message, Uri brokerServiceName = null);
+
         /// <summary>
         /// Publish a message.
         /// </summary>
         /// <param name="service"></param>
         /// <param name="message"></param>
         /// <param name="brokerServiceName">The name of a SF Service of type <see cref="BrokerService"/>.</param>
         /// <returns></returns>
-        Task PublishMessageAsync(StatelessService service, object message,
-            Uri brokerServiceName = null);
+        Task PublishMessageAsync(StatelessService service, object message, Uri brokerServiceName = null);
```

Comment:
The original 2 methods take a service object, but never uses it.  Is there a reason for this?  I kept these methods around for backwards compatibility, but they look like they can be removed at some point, probably at the next major version bump.  What are your thoughts on this @loekd?

Reply:
I believe you found a bug. The broker service can be inside different applications, so original code (the extension methods) use the service to get the application:
```
            if (brokerServiceName == null)
            {
                brokerServiceName =
                    await
                        DiscoverBrokerServiceNameAsync(
                            new Uri(service.Context.CodePackageActivationContext.ApplicationName));
                if (brokerServiceName == null)
                {
                    throw new InvalidOperationException(
                        "No brokerServiceName was provided or discovered in the current application.");
                }
            }
```